src/cromshell/utilities/cromshellconfig.py

```diff
@@ -23,6 +24,25 @@
 submission_file = None
 cromshell_config_options = None
 cromwell_server = None
+# Request defaults
+requests_connect_timeout = 5
+requests_verify_certs = True
+
+
+def override_requests_parameters(server_timeout: int, skip_certs: bool):
+    """Override requests settings for timeout and certs verification"""
+
+    global requests_connect_timeout
+    global requests_verify_certs
+
+    if server_timeout:
+        requests_connect_timeout = server_timeout
+
+    if skip_certs is True:
+        requests_verify_certs = False
+        # Hide requests warning about not verifying certs.
+        warnings.filterwarnings('ignore',
```

Comment:
I'm not following why the warning is suppressed.

Reply:
A message about skipping certs is passed to the loger, so any future warning from requests about this is hidden.